unit_tests/test_latte_thread.py

```diff
@@ -120,6 +121,11 @@ def test_04_latte_run_client_encrypt(request, docker_scylla, params):
 
     loader_set = LocalLoaderSetDummy(params=params)
 
+    # dedicated SSL certs directory for the test, to avoid conflicts during parallel tests execution
+    if ssl_dir := request.node.stash.get(ssl_dir_key, None):
+        for loader_node in loader_set.nodes:
+            loader_node.__class__.ssl_conf_dir = property(lambda self: ssl_dir)
```

Comment:
Cannot we pass it like a parameter to LocalLoaderSetDummy? It is a testing object, so we can easily change it